src/app/screens/LNURLAuth/index.tsx

```diff
@@ -1,140 +1,30 @@
-import Button from "@components/Button";
-import ConfirmOrCancel from "@components/ConfirmOrCancel";
-import Container from "@components/Container";
-import ContentMessage from "@components/ContentMessage";
-import PublisherCard from "@components/PublisherCard";
-import ResultCard from "@components/ResultCard";
-import React, { useState } from "react";
-import { useTranslation } from "react-i18next";
-import { useNavigate } from "react-router-dom";
-import ScreenHeader from "~/app/components/ScreenHeader";
-import { useNavigationState } from "~/app/hooks/useNavigationState";
-import { USER_REJECTED_ERROR } from "~/common/constants";
+import { useEffect, useState } from "react";
+import LNURLAuthComponent from "~/app/components/LNURLAuth";
+import Onboard from "~/app/components/onboard";
+import { useAccount } from "~/app/context/AccountContext";
 import api from "~/common/lib/api";
-import msg from "~/common/lib/msg";
-import type { LNURLAuthServiceResponse } from "~/types";
 
-function LNURLAuth() {
-  const { t } = useTranslation("translation", { keyPrefix: "lnurlauth" });
-  const { t: tCommon } = useTranslation("common");
+export default function LNURLAuth() {
+  const { account } = useAccount();
+  const [hasMnemonic, setHasMnemonic] = useState(false);
 
-  const navigate = useNavigate();
-  const navState = useNavigationState();
+  useEffect(() => {
+    async function fetchAccountAndSetComponent() {
```

Comment:
Note: this method is not named what it actually does. Although this code probably has to change

Reply:
resolved